### PR TITLE
change readme to point to coredns helm repo and other doc fixes

### PR DESCRIPTION
**Pull request by ajayk**

### CODE REVIEW

File: stable/coredns/README.md [hunk 1]
```diff
@@ -47,7 +48,7 @@ The command removes all the Kubernetes components associated with the chart and
 | Parameter                               | Description                                                                           | Default                                                     |
 |:----------------------------------------|:--------------------------------------------------------------------------------------|:------------------------------------------------------------|
 | `image.repository`                      | The image repository to pull from                                                     | coredns/coredns                                             |
-| `image.tag`                             | The image tag to pull from                                                            | `v1.7.1`                                                    |
+| `image.tag`                             | The image tag to pull from                                                            | `1.8.0`                                                    |
```
Comment (haad) [inline stable/coredns/README.md:51]: This is not true. Can you remove it ?

Comment (ajayk) [reply stable/coredns/README.md:51]: @haad Both the charts which are tagged have 1.8.0 as default in the `values.yaml ` Not sure where the v1.7.1is based out on this repository

**Comment by haad** [comment]
Until you finish DCO we can't merge this.

**Comment by ajayk** [comment]
> Until you finish DCO we can't merge this.

Completed the DCO

**Pull request merged**